pkg/alertmanager/alertmanager_ring.go

```diff
@@ -73,6 +77,23 @@ func (cfg *RingConfig) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	cfg.RingCheckPeriod = 5 * time.Second
 }
 
+// NewRing creates a new client ring for the clientComponent, with servicediscovery notification being sent to the receiver.
+func NewRing(cfg RingConfig, clientComponent string, receiver servicediscovery.Notifications, logger log.Logger, reg prometheus.Registerer) (services.Service, error) {
+	// Since this is a client for the alert-managers ring, we append "alert-manager-client" to the component to clearly differentiate it.
+	clientComponent += "-alert-manager-client"
```

Comment:
In the config please write "alertmanager" instead of "alert-manager" to keep consistency.

```suggestion
	clientComponent += "-alertmanager-client"
```

Reply:
_Same comment applies everywhere else._